### ISSUE TITLE
RuntimeError: Expected ASGI message 'websocket.accept', 'websocket.close', or 'websocket.http.response.start' but got 'http.response.start'.

### ISSUE THREAD

**Issue by wielandb** [zauberzeug/nicegui#2368]
### Description

I am experiencing an issue similar to #1467.

A working nicegui project worked fine, but after upgrading nicegui to the latest version I get the following error:

```
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\uvicorn\protocols\websockets\websockets_impl.py", line 250, in run_asgi
    result = await self.app(self.scope, self.asgi_receive, self.asgi_send)
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\uvicorn\middleware\proxy_headers.py", line 84, in __call__
    return await self.app(scope, receive, send)
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\fastapi\applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\starlette\applications.py", line 123, in __call__
    await self.middleware_stack(scope, receive, send)
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\starlette\middleware\errors.py", line 151, in __call__
    await self.app(scope, receive, send)
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\starlette\middleware\base.py", line 103, in __call__
    await self.app(scope, receive, send)
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\starlette\middleware\gzip.py", line 26, in __call__
    await self.app(scope, receive, send)
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\starlette\middleware\exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\starlette\_exception_handler.py", line 64, in wrapped_app
    raise exc
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    await app(scope, receive, sender)
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\starlette\routing.py", line 762, in __call__
    await self.middleware_stack(scope, receive, send)
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\starlette\routing.py", line 782, in app
    await route.handle(scope, receive, send)
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\starlette\routing.py", line 485, in handle
    await self.app(scope, receive, send)
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\engineio\async_drivers\asgi.py", line 69, in __call__
    await self.not_found(receive, send)
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\engineio\async_drivers\asgi.py", line 117, in not_found
    await send({'type': 'http.response.start',
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\starlette\_exception_handler.py", line 50, in sender
    await send(message)
  File "C:\Users\me\AppData\Local\Programs\Python\Python310\lib\site-packages\uvicorn\protocols\websockets\websockets_impl.py", line 330, in asgi_send
    raise RuntimeError(msg % message_type)
RuntimeError: Expected ASGI message 'websocket.accept', 'websocket.close', or 'websocket.http.response.start' but got 'http.response.start'.
```

My python-engineio version is 4.8.2., so the problem in the linked issue that was fixed by upgrading from 4.8.0 does not seem to be the problem.

**Comment by petergaultney** [comment]
I think this is basically the same as #2354 . It happened to me, too, after upgrading some packages (actually because of a Python upgrade in my case).

try the workaround listed here: https://github.com/zauberzeug/nicegui/issues/2354#issuecomment-1888139684  It worked for me.

**Comment by rodja** [comment]
We just released 1.4.10 which fixed the FastAPI incompatibility. Please give it a try.

**Comment by wielandb** [comment]
It works after upgrading. Thank you. 🙂
